docs/docs/running-hello-world.md

```diff
@@ -73,216 +75,235 @@ The web server loads the enclave and waits for requests from the client. Amongst
-  # A constraint which defines which enclaves the client will accept.
-  "S:0000000000000000000000000000000000000000000000000000000000000000 PROD:1 SEC:INSECURE" \
-  # The string to reverse
-  reverse-me
+cd conclave-tutorials/hello-world
 ```
+3. Run the client with the command:
+```bash
+java -jar client/build/libs/client.jar "S:0000000000000000000000000000000000000000000000000000000000000000 PROD:1
+SEC:INSECURE" reverse-me
+```
+In the above command, the parameters in quotes are the constraints, and 'reverse-me' is the string to be reversed.
 
-You should see the output
+You will see the output:
 ```bash
 Reversing `reverse-me` gives `em-esrever`
 ```
-which has been produced by the following sequence of events:
-
-1. The host loaded the enclave, which generated a remote attestation report.
-2. The client connected to the host and downloaded the attestation report.
-3. The client used the attestation report to verify that it was connected to the enclave it expected, and that the
-   enclave was running in the expected mode.
-4. The client encrypted the "reverse-me" string using the public key provided in the attestation.
-5. As part of the message to the enclave, the client also included a public key for the enclave to use when responding.
-6. The client sent the encrypted message to the host's REST API.
-7. The host passed the encrypted message to the enclave.
-8. The enclave decrypted the client's message, reversed the string, and encrypted the response using the client's
+
+You have run your first enclave successfully.
+
+The following sequence of events produces the above output:
+
+1. The host loads the enclave, which generates a remote attestation report.
+2. The client connects to the host and downloads the attestation report.
+3. The client uses the attestation report to verify that it is connected to the enclave it expected, and that the
+   enclave is running in the expected mode.
+4. The client encrypts the "reverse-me" string using the public key provided in the attestation.
+5. In the message to the enclave, the client also includes a public key for the enclave to use when responding.
+6. The client sends the encrypted message to the host's REST API.
+7. The host passes the encrypted message to the enclave.
+8. The enclave decrypts the client's message, reverses the string, and encrypts the response using the client's
    public key.
-9. The enclave passed the encrypted message to the host, was sent back to the client as an HTTP response to the same
+9. The enclave passes the encrypted message to the host, and is sent back to the client as an HTTP response to the same
    request.
 
-Congratulations on running your first enclave!
-
 ### Enclave Constraint
-The `--constraint` parameter of the client command-line interface defines the 
-properties of an acceptable enclave. This includes the signing key 
+The `--constraint` parameter of the client defines the properties of an acceptable enclave.
```

Comment:
We actually don't use `--constraint` in this example anymore. I would just say _The constraint parameter of the client_

Reply:
Done.

Reply:
I removed this section to declutter the tutorial, as part of another comment.